src/cc65/symentry.h

```diff
@@ -245,10 +246,10 @@ INLINE int SymIsRegVar (const SymEntry* Sym)
 /* Return true if the given entry is a register variable */
 /* ### HACK! Fix the ugly type flags! */
 {
-    return ((Sym->Flags & (SC_REGISTER|SC_TYPE)) == SC_REGISTER);
+    return ((Sym->Flags & (SC_REGISTER|SC_TYPEMASK)) == SC_REGISTER);
 }
 #else
-#  define SymIsRegVar(Sym)      (((Sym)->Flags & (SC_REGISTER|SC_TYPE)) == SC_REGISTER)
+#  define SymIsRegVar(Sym)  (((Sym)->Flags & (SC_REGISTER|SC_TYPEMASK)) == SC_REGISTER)
```

Comment:
It seems there should have been blanks here before, shouldn't they?

Reply:
Fixed.